CONTRIBUTING.md

```diff
@@ -186,12 +186,11 @@ issue](https://github.com/open-telemetry/opentelemetry-specification/issues/165)
 
 ## Style Guide
 
-This project includes a
-[`.editorconfig`](https://github.com/open-telemetry/opentelemetry-dotnet/blob/master/.editorconfig)
-file which is supported by all the IDEs/editor mentioned above. It works with
-the IDE/editor only and does not affect the actual build of the project.
+This project includes a [`.editorconfig`](./.editorconfig) file which is
+supported by all the IDEs/editor mentioned above. It works with the IDE/editor
+only and does not affect the actual build of the project.
 
 This repository also includes [stylecop ruleset
-files](https://github.com/open-telemetry/opentelemetry-dotnet/tree/master/build).
+files](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/build).
```

Comment:
relative link?

Reply:
docfx doesn't support relative link to a folder (it only supports link to a file) - so if we change it to relative, we'll get a build error.

Reply:
I could change this to one file (e.g. `OpenTelemetry.prod.ruleset`) if it sounds okay.

Reply:
I just changed it to a text instead of link and it seems better.